README.rst

```diff
@@ -1,6 +1,29 @@
-This is the core repository of the Open edX software. It includes the LMS
-(student-facing, delivering courseware), and Studio (course authoring)
-components.
+#################
+Open edX Platform
+#################
+| |License: AGPL v3| |Status| |Python CI|
+
+.. |License: AGPL v3| image:: https://img.shields.io/badge/License-AGPL_v3-blue.svg
+  :target: https://www.gnu.org/licenses/agpl-3.0
+
+.. |Python CI| image:: https://github.com/openedx/edx-platform/actions/workflows/ci.yml/badge.svg
+  :target: https://github.com/openedx/edx-platform/actions/workflows/ci.yml
+
+.. |Status| image:: https://img.shields.io/badge/status-maintained-31c653
+
+Purpose
+-------
+The `Open edX Platform <https://openedx.org>`_ is a service-oriented platform for authoring and
+delivering online learning at any scale.  The platform is written in
+Python and JavaScript and makes extensive use of the Django
+framework. At the highest level, the platform is composed of a
+monolith, some independently deployable applications (IDAs), and
+micro-frontends (MFEs) based on the ReactJS.
+
+This repository hosts the monolith at the center of the Open edX
+platform.  Functionally, the edx-platform repository provides two services:
```

Comment:
Whoops, sorry @e0d , one more thing -- you need a newline here for the RST bullet points to render.

```suggestion
platform.  Functionally, the edx-platform repository provides two services:

```